### PR TITLE
Base of All Items/Pickups Created.

### PR DESCRIPTION
**Pull request by Jeff-Moreau**
Need to add some functionality but most should be there.
Leaf
Grub
Eucalyptus
Green Bud
Blue Bud
Red Bud

### CODE REVIEW

**Review by Guysus — APPROVED** [review]
looks good

**Review by Saretzky — CHANGES_REQUESTED** [review]
Should the cutscene and skybox stuff be there?

File: Source/SteveSloth/Private/ItemBaseClass.cpp [hunk 1]
```diff
@@ -1,37 +1,86 @@
...
+		Name = itemData->Name;
+		Mesh = itemData->Mesh;
+		Health = itemData->Health;
+		ItemType = itemData->ItemType;
+		DropChance = itemData->DropChance;
+		StackAmount = itemData->StackAmount;
+		bIsCurrency = itemData->bIsCurrency;
+		AddHealthAmount = itemData->AddHealthAmount;
+		AddHealthPercentage = itemData->AddHealthPercentage;
+	}
```
Comment (Saretzky) [inline Source/SteveSloth/Private/ItemBaseClass.cpp:33]: figured out this should be in the begin play (line 20 to 33)

**Comment by Jeff-Moreau** [comment]
> Should the cutscene and skybox stuff be there?

Ya I messed up the why I updated from Main.. So that is the Update from main stuff, just happened differently

**Review by Saretzky — CHANGES_REQUESTED** [review]

**Review by Saretzky — APPROVED** [review]
once conflicts done you're good to go

**Pull request merged**